app/assets/javascripts/components/input_table.ts

```
@@ -0,0 +1,244 @@
+        const response = await fetch(this.route, {
+            method: "PATCH",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify({
+                score_items: this.editedScoreItems
+            })
+        });
+        if (response.ok) {
+            const js = await response.text();
+            eval(js);
```

Biome

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)